requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.28.1

```

Comment:
use `pip freeze > requirements.txt` to save versions for all dependencies

Reply:
But it works, if I add more requirements, the image will be bigger, right?